src/test/resources/TestNG.xml

```diff
@@ -14,6 +15,9 @@
             <class name="PlaySongByActionsTest"/>
             <class name="RenamePlayListTest"/>
             <class name="ThemeTest"/>
+            <class name="Homework17"/>
+            <class name="Hw18"/>
+            <class name="hw19"/>
```

Comment:
Add this in TestNG for run tests

For example

![image](https://github.com/Dzzmitryy/A52-QA-Automation-Framework/assets/104761482/b41fc88d-985c-4582-b796-1b82daff7eb0)